### ISSUE TITLE
Missing sample files

### ISSUE THREAD

**Issue by leus** [arthg/Arthware.X937Parser#6]
Hey, nice project. I'm researching into this and there is a serious lack of open source efforts in this area; and the commercial solutions are just abysmal.

I cloned the project, upgraded it to netcore 3.1 (no changes needed) but the tests fail because of missing files:

```
System.IO.FileNotFoundException : Could not find file 'd:\Repos\Arthware.X937Parser\Arthware.X937Parser\Arthware.X937Parser.Tests.Integration\bin\Debug\netcoreapp3.1\SampleX9ReturnFiles\BT302IMGBO.CACHETRETURN.20170208123227.x937'.
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Arthware.X937Parser.Tests.Integration.X937ParserTests.GetX937ReturnsMethod.Should_emit_all_the_returns_when_multiple_returns_are_present_on_the_X9_file() in d:\Repos\Arthware.X937Parser\Arthware.X937Parser\Arthware.X937Parser.Tests.Integration\X937ParserTests.cs:line 43
```

Any chance you could add some sample files? (I don't have any)

**Comment by arthg** [comment]
@leus Thank you for the feedback about the project.  It has been a few years since I contributed the code and there has not been much interest.  (ok, no interest)
So I am relying on my memory about the decision not to include the missing sample file.  I probably intentionally omitted the file was because of uncertainty about whether I can share the contents publicly.  I did locate the file and one other on one of my older laptops.  I will locate an X9 viewer and review the contents to assess whether I can add them in the repository.
As I recall, I had the same challenge to locate sample files for testing.

**Comment by arthg** [comment]
@leus I downloaded the Mavro X9 viewer and discovered that indeed the sample X9 file referenced in the project integration test includes 2 returned checks with visible MICR line.  It would not be appropriate to share the file publicly.  In fact I just now noticed my comment in the  code:
`
 //this file  not included in version control:  may contain production information
`
I was able to locate only 1 other sample file and it also contains a real check and MICR.  Sorry but I will not be able to add any sample files.
